tools/halide_image_io.h

```diff
@@ -93,6 +95,8 @@ inline void swap_endian_16(bool little_endian, uint16_t &value) {
     }
 }
 
+#define FILEOPENER_MAX_LINE_LEN 1024
```

Comment:
Please use constexpr for constants; preprocessor macros are best avoided unless there is no alternative